### DISCUSSION TITLE
Add setting similar to python.testing.autoTestDiscoverOnSaveEnabled

### DISCUSSION THREAD

**Discussion by jivko-chobanov** [elixir-lsp/vscode-elixir-ls#441] [Ideas]
I would like to be able to disable either:
- the auto test discovery or
- TestExplorer in general or
- the auto run test on test file save.

It's because it is constantly using my CPU and slowing down my workflow. I prefer to run tests in terminal (where I can Ctrl+click the file path in the stacktrace to open it).

I saw a similar setting in Python extension called - python.testing.autoTestDiscoverOnSaveEnabled.

**Comment by lukaszsamson** [comment]
There is no auto run tests on save. The test discovery only compiles test files. If your test suite starts running as a side effect on compile then you should address that instead of disabling the discovery.

**Reply by jivko-chobanov** [reply]
I agree - it seems to be only compilation, but it happens on every change even without saving the test file. I have integration tests that are very long files (e.g. 3500 lines) and it takes several (sometimes 10) seconds for such file to compile or whatever it does when it shows an icon of circling snake in front of the file in TestExplorer.
